bin/test

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+
+# fail hard
+set -eo pipefail
+# fail harder
+set -eu
+
+
+# name the command line argument for this script
+declare build_dir="${1:-}"
+
+# check if composer tests are available
+if cd "$build_dir" && composer run-script -l 2>/dev/null | grep -q '^\s*test\b'; then
```

Comment:
This could just be a separate:

```
pushd "$build_dir" > /dev/null
```

So you don't have to cd in the if statement.

Reply:
Thank you for this hint! I think it's better to leave the pushd in the if-statement in order to check whether the script runs the test-command in an existing directory or not.